addons/website/static/src/js/content/menu.js

```diff
@@ -603,6 +603,19 @@ publicWidget.registry.hoverableDropdown = animations.Animation.extend({
             this.$dropdownMenus.css('top', '');
         }
     },
+    /**
+     * @private
+     */
+    _toggleDropdown: function (ev) {
+        let { currentTarget } = ev;
+        let dropdownToggle = currentTarget.querySelector('.dropdown-toggle');
+        if (config.device.size_class <= config.device.SIZES.SM ||
+            currentTarget.classList.contains('o_extra_menu_items') ||
+            !dropdownToggle) {
```

Comment:
Instead of
```javascript
a = f();
if (X || Y || a) {
    return;
}
a.doStuff();
```
do
```javascript
if (X || Y) {
    return;
}
a = f();
if (a) {
    return;
}
a.doStuff();
```

it's a micro-optimization so not important in terms of perf in this specific case I guess but I feel it's important to just do that systematically. Also use `const` instead of `let`.